index.bs

```diff
@@ -1567,11 +1564,11 @@ dictionary MessageArgs {
 
 ### SIMID:Creative:requestChangeAdDuration ### {#simid-creative-requestChangeAdDuration}
 
-See <a href="#diagram-durationchange-known">Diagram 11. Known Ad Duration Change Sequence</a> and <a href="#diagram-durationchange-unknown">Diagram 12. Unknown Ad Duration Change Sequence</a>.
+See <a href="#diagram-durationchange-known">Diagram 10. Known Ad Duration Change Sequence</a> and <a href="#diagram-durationchange-unknown">Diagram 11. Unknown Ad Duration Change Sequence</a>.
 
 In SIMID, ad's media determines the initial ad duration. The ad span may change due to user interaction or other factors. When ad duration changes, the creative posts `Creative:requestChangeAdDuration` message that communicates an updated value. In response to the `requestChangeAdDuration` message, the player adjusts ad-end timing and updates its ad progress UI (eg., countdown).
 
-The creative expresses a known duration value in seconds. In cases the duration is unknown (typically due to user interaction), the value is `-2`. With a known duration, the player unloads the ad automatically once the countdown (ad remaining time) reaches zero (<a href="#diagram-durationchange-known">diagram 11</a>,  <span steps><b>4</b><b>5</b><b>6</b></span>). With the duration value `-2`, the player displays the ad indefinitely until the creative posts [[#simid-creative-requestStop]] (<a href="#diagram-durationchange-unknown">diagram 12</a>,  <span steps><b>4</b></span>). 
+The creative expresses a known duration value in seconds. In cases the duration is unknown (typically due to user interaction), the value is `-2`. With a known duration, the player unloads the ad automatically once the countdown (ad remaining time) reaches zero (<a href="#diagram-durationchange-known">diagram 10</a>,  <span steps><b>4</b><b>5</b><b>6</b></span>). With the duration value `-2`, the player displays the ad indefinitely until the creative posts [[#simid-creative-requestStop]] (<a href="#diagram-durationchange-unknown">diagram 11</a>,  <span step><b>4</b></span>). 
```

Comment:
Either:
In case the duration is unknown
Or:
In cases where the duration is unknown